bypass_url_parser.py

```diff
@@ -304,6 +306,29 @@ def _generate_curls(self, url_obj: ParseResult):
                     if item not in self.curl_items:
                         self.curl_items.append(item)
 
+        # [end_paths] - Add suffix
+        if "all" in self.current_bypass_modes or "end_paths" in self.current_bypass_modes:
+            commands = set()
```

Comment:
Maybe we could do this without looping on commands but adding CurlItem right-away?